console/src/main/scripts/plugins/accounts/ts/accountsGlobals.ts

```diff
@@ -60,9 +60,13 @@ module HawkularAccounts {
   }
 
   export interface IOrganizationMembership {
+    id: string;
     organization: IOrganization;
     member: IPersona;
     role: IRole;
+    $update(options:{},
+            success?:(success:ISuccessPayload) => void,
```

Comment:
For the current implementation, it's not needed, as we are not reading results from the server. I think other operations indeed have a more specialized type, but I don't think it's needed for this specific operation.